src/plonk.rs

```diff
@@ -8,6 +8,7 @@
 use blake2b_simd::Params as Blake2bParams;
 
 use crate::arithmetic::{CurveAffine, FieldExt};
+use crate::helpers::CurveRead;
```

Comment:
This lints as unused because `CurveAffine` also imports a `read` function. In a subsequent PR we'll update to a version of `pasta_curves` where `CurveAffine::read` is removed.